data/civ.yaml

```
@@ -0,0 +1,6 @@
+name: Côte d'Ivoire
+official_name: 	The Republic of Côte d'Ivoire
```

Can you remove the extra spaces, please?
```suggestion
official_name: The Republic of Côte d'Ivoire
```

---

Am on it